src/sentry/api/bases/organization.py

```diff
@@ -344,11 +344,7 @@ def get_projects(self, request, organization, project_ids=None):
             return []
 
         return super(OrganizationReleasesBaseEndpoint, self).get_projects(
-            request,
-            organization,
-            force_global_perms=has_valid_api_key,
-            include_all_accessible=True,
```

Comment:
This looks like the only place we were using this parameter. Could we remove it entirely? There is a note that it should be removed.

Reply:
As far as I can tell, you're right, there's no reason to keep it around :+1: